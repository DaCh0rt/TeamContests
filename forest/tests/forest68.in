304 275
-6712 2441
-5985 -641
2424 5116
3207 843
-7647 -8109
-1803 -3922
-3360 8844
3013 3946
572 -7554
6487 -2342
-4111 2395
5518 -6500
5028 2032
-2481 -1963
3696 5796
-8273 -7188
-3486 -6536
-472 -7487
-7306 -7867
3971 -8597
2486 1955
-5593 3318
-3239 -5185
8231 8985
1257 -6516
495 4152
197 -2369
4424 5056
8386 -4243
7759 2950
2847 -7089
-335 -506
-8289 9104
4793 -5786
7752 5501
2190 -7908
9522 3529
-2571 -6702
5207 8572
-569 -6897
-9157 -3235
-7319 6406
-2999 5959
-5448 5468
1481 6031
-5374 -656
9787 434
-8985 -4369
-7778 -5221
1495 4963
-9153 8422
-7555 3481
-8613 -8651
2870 -3582
2433 3277
-668 4713
-7519 -2818
6216 8894
2534 -8180
1036 1307
2794 -5922
8674 7379
-6091 -8947
-6984 -2692
91 -4498
-6016 -8013
2635 -8894
8671 5830
2876 1455
7753 -1888
-1921 9269
1828 -8075
-7258 -5619
4885 1037
-2073 7242
-3719 1350
8263 -6420
806 -7152
-5095 -8956
820 8548
9261 -8426
-4284 -3314
-9178 2458
-401 -3766
-9275 -8020
-9963 -3962
7910 7550
6570 -6905
-8518 4336
-1468 -4417
6685 6837
2737 -7741
5017 3696
-9915 -5646
-4647 -7521
-2481 6
-2266 -6175
8163 3067
6240 2391
-6410 8442
-9234 8784
-9326 614
-7658 118
-7486 -1542
1624 996
5392 -4621
-2001 -1888
3750 -9627
9585 5327
6084 -7799
-7614 -4945
2289 -6586
-3948 5967
-8438 -9137
-1306 51
942 -2535
7425 862
1310 -5141
914 2958
-5554 -7036
9065 2251
9559 -2340
-1006 6901
7486 -6381
-2224 2421
-3595 3265
-9850 1845
-2504 -9160
5669 -4771
-5000 1301
-6648 6913
-6249 2881
-823 -6738
9567 6112
-2221 9146
-9486 2965
-3031 -9009
1397 -8476
8291 7948
2394 3539
6428 8794
1935 6581
2297 3996
3533 -2946
3052 9781
853 907
2395 6396
9857 -1121
-2260 -1371
-4724 5082
-2739 7469
7754 2937
-2338 -1820
-6270 3284
-7714 7344
8259 -251
9863 2516
-3313 1162
-5559 1706
9287 -9631
4495 -6957
7653 4078
3895 1138
-8734 7789
3699 -2144
-5189 932
-3264 7178
1843 8891
9523 9840
-2756 -6177
-7326 -4509
-2206 -9503
5185 -4073
-9575 3871
9587 8967
6840 1286
-3200 1714
8457 -7016
4955 -7549
-6586 -5064
-7891 -877
7709 -1105
4266 5387
7835 4397
-9970 -3666
4883 -6922
386 4719
-8870 5787
-1534 -802
-7753 5837
-3996 5888
7443 2995
3325 -767
-2980 4601
8022 -1179
1766 -5970
9125 2158
6875 2
-38 8889
-7758 6117
-9173 4501
5807 -6421
3845 955
6404 -231
5566 6103
-5447 9634
8980 -9022
7528 -4870
-8583 -8623
2295 -2356
4294 6004
-7092 -6753
-755 -5322
4228 -5087
3889 -9832
-8524 -8770
3152 -2857
8394 7383
-2340 1949
-6983 9678
-1815 8348
-1434 -72
4003 -2545
4969 -6654
-5691 -5646
-8136 -6526
8341 -9556
8165 -614
-7648 4730
6482 5474
-7700 -1131
8110 -5488
-6805 3754
3419 -385
6145 -7682
-7612 -5101
4933 -2968
2111 4923
8586 6148
9009 -4831
8399 -4710
8243 5164
-3121 2096
-3274 7910
4223 -8975
-9138 -8524
-8461 -8297
4142 -2544
5052 9612
-5404 -7368
521 3688
-2901 -6671
-3506 1335
7364 -9411
-5331 2866
1417 4875
3781 6499
2472 6111
-520 -5848
4134 5813
8895 -5453
598 -7226
2071 -6947
-5832 3849
4856 -2246
2096 -2354
4916 3473
-2749 -8702
-3171 7752
-9298 -5102
-9248 1188
9612 -4443
-6236 7215
-1134 3769
-1020 9922
84 -9156
-8691 5866
3314 8242
1453 6766
-2971 -4599
3566 3376
7612 6804
-5272 -6688
2387 8006
-9679 -5027
1313 -1320
1719 4813
7732 3782
-9974 5247
6250 -1053
5756 8054
-4499 9719
-7651 2754
5207 -7200
365 1283
-1746 -9481
3848 -8849
-4818 -8891
-147 2578
-7009 4029
-7667 -4371
2078 -3018
3174 -1366
-8786 3827
