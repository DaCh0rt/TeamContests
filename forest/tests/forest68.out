76326.74 396060270.39
